### ISSUE TITLE
(short issue description)

### ISSUE THREAD

**Issue by chiefMarlin** [NVIDIA/aistore#180]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Hi,
It looks like there is an issue with the entrypoint.sh script in the Dockerfile, documented here https://github.com/NVIDIA/aistore/blob/main/deploy/prod/docker/single/README.md

Container exits with an error as per below:

```bash
root@host:~# docker run -d   -p 51080:51080 -v $(mktemp -d):/ais/disk0   aistorage/cluster-minimal:latest
85796b786a986ea4571a283f081fdfc3002a0df840f269601177974cd345c97c
```
```bash
root@host:~# docker ps -a
CONTAINER ID   IMAGE                              COMMAND                  CREATED         STATUS                              PORTS     NAMES
85796b786a98   aistorage/cluster-minimal:latest   "sh -c 'entrypoint/e…"   5 seconds ago   Exited (0) Less than a second ago             silly_cartwright
```

```bash
root@host:~# docker logs silly_cartwright
aisnode_config.sh: line 4: make_backend_conf: command not found
E 21:29:52.236893 daemon:154 FATAL ERROR: failed to load initial global config "/etc/aisnode/proxy/ais.json": cmn.ClusterConfig.Backend: Skip: do not know how to skip: 44, error found in #10 byte of ...|ackend": ,
	"mirror"|..., bigger context ...|{
	"backend": ,
	"mirror": {
		"copies":       2,
		"burst_buffer|...

aisnode_config.sh: line 4: make_backend_conf: command not found
```

## If i am not mistaken, its looking for the function that is here: https://github.com/NVIDIA/aistore/blob/4471151595cb8e27246887c25a33ca2642fdea64/deploy/dev/utils.sh#L112

```bash
make_backend_conf() {
  backend_conf=()
  for backend in ${AIS_BACKEND_PROVIDERS}; do
    case $backend in
      aws)   backend_conf+=('"aws":   {}') ;;
      azure) backend_conf+=('"azure": {}') ;;
      gcp)   backend_conf+=('"gcp":   {}') ;;
      ht)    backend_conf+=('"ht":    {}') ;;
    esac
  done
  echo {$(IFS=$','; echo "${backend_conf[*]}")}
}
```

### Expected Behavior

Container should stay running as per guide.

### Current Behavior

Container exits with an error described in first section.

### Steps To Reproduce

Steps to reproduce are outlines in the first section.

### Possible Solution

Suggested fix is outlined in the first section.

### AIStore build/version

Docker image id 858497ea0e9a

### Environment details (OS name and version, etc.)

Linux/Docker/Image: 858497ea0e9a

**Comment by chiefMarlin** [comment]
After re-pulling a fresh image aistorage/cluster-minimal:latest its working properly now.

**Comment by alex-aizman** [comment]
yes, fixed yesterday by this commit: 3c0a49a1ec584a221a91f54ef828d091dbf4dc85
